portal/models/identifier.py

```diff
@@ -40,6 +40,11 @@ def value(self, value):
 
     @classmethod
     def from_fhir(cls, data):
+        if not data or not all((data.get('system'), data.get('value'))):
+            raise ValueError(
+                "Ill formed 'identifier'; requires both 'system' and "
```

Comment:
I don't think this requirement is strictly necessary (resources that break it may still be FHIR compliant).

Do we have partial QuestionnaireResponses lacking `identifier`s?

Reply:
No partials.  Just all the submissions from WiserCare (where they're submitting the patient identifier w/o a system).